.changeset/spotty-kangaroos-shout.md

```diff
@@ -0,0 +1,5 @@
+---
+"@viron/lib": patch
+---
+
+Set patition option is false by default
```

Comment:
Could you fix a typo of the word partition?

"Set the partition option to false as the default."